packages/generator/shared.js

```diff
@@ -0,0 +1,570 @@
+const spawnAsync = require('@expo/spawn-async');
+const fetch = require('cross-fetch');
+const ejs = require('ejs');
+const fs = require('fs');
+const fsExtra = require('fs-extra');
+const path = require('path');
+const prettier = require('prettier');
+
+// eslint-disable-next-line import/order
+const contributors = require('./contributors');
```

Comment:
```suggestion
const contributors = require('./contributors');
const PackageVersion = require('../../package.json').version;
```

You can move the remaining require package.json file here so that eslint will be happy